doc/models/paypal-experience-landing-page.md

```diff
@@ -1,11 +1,11 @@
 
-# Pay Pal Experience Landing Page
+# Paypal Experience Landing Page
```

Comment:
All of these titles are a little funny now. I get that it's just matching the programming model name, but in that case, I don't even know why there are spaces.

Reply:
TBF, they were funny before with the spaces, too, so it's an improvement!